storage/filesystem/internal/dotgit/dotgit.go

```diff
@@ -103,3 +106,70 @@ func (d *DotGit) Idxfile() (fs.FS, string, error) {
 
 	return nil, "", ErrIdxNotFound
 }
+
+// Objectfiles returns the hash slice of the objects files in the "objects/"
```

Comment:
Grammar improvement: Objectfiles returns a slice with the hashes of objects found under the .git/objects/ directory.